packages/core/src/utils/JsonTransformer.ts

```diff
@@ -1,15 +1,41 @@
+import type { Validate } from 'class-validator'
+
 import { instanceToPlain, plainToInstance, instanceToInstance } from 'class-transformer'
 
+import { ClassValidationError } from '../error/ClassValidationError'
+
+import { MessageValidator } from './MessageValidator'
+
+interface Validate {
+  validate?: boolean
+}
+
 export class JsonTransformer {
   public static toJSON<T>(classInstance: T) {
     return instanceToPlain(classInstance, {
       exposeDefaultValues: true,
     })
   }
 
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  public static fromJSON<T>(json: any, Class: { new (...args: any[]): T }): T {
-    return plainToInstance(Class, json, { exposeDefaultValues: true })
+  public static fromJSON<T>(
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    json: any,
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    cls: { new (...args: any[]): T },
+    { validate }: Validate = { validate: true }
```

Comment:
If we set it right from the `=`, it means if I pass an empty object to  (`fromJson(json, Class, {})`) it will be set to false. Setting it left of the `=` means it will be assigned whenever the property `validate` is undefined.

```suggestion
    { validate = true }: Validate = {  }
```

Reply:
ah sorry already sorted - haven't pushed